bm/charter_condition.go

```diff
@@ -0,0 +1,47 @@
+package bm
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"time"
+)
+
+// CharterCondition response description
+type CharterCondition struct {
+	Period          *Period   `json:"period,omitempty"`
+	CheckInDays     []*int    `json:"checkInDays,omitempty"`
+	CheckOutDays    []*int    `json:"checkOutDays,omitempty"`
+	MinimumDuration int       `json:"minimumDuration,omitempty"`
+	Overnight       bool      `json:"overnight,omitempty"`
+	CheckInTime     time.Time `json:"defaultCheckInTime,omitempty"`
+	CheckOutTime    time.Time `json:"defaultCheckOutTime,omitempty"`
+	LicenseRequired bool      `json:"licenseRequired,omitempty"`
+	SkipperRequired bool      `json:"requiredSkipperLicense,omitempty"`
+}
+
+type CharterConditionService service
+
+// GetCharterCondition
+func (os *CharterConditionService) GetCharterCondition(id int) (or *Yatch, err error) {
```

Comment:
Since this is returning a Yacht object, I'd much rather have it handle yacht requests.

So something like GetYacht or such. I'd also put the related methods to fetching a yacht in the yacht file.

Could you also fix this spelling mistake everwhere 😂  Driving me up the walls.